commit 1f3ef0efbacb2aa63e8e3933664192ee27d0d95b
Author: Guchun Chen <guchun.chen@amd.com>
Date:   Fri Apr 10 11:41:54 2020 +0800

    drm/amdgpu: add uncorrectable error count print in UMC ecc irq cb
    
    Uncorrectable error count printing is missed when issuing UMC
    UE injection. When going to the error count log function in GPU
    recover work thread, there is no chance to get correct error count
    value by last error injection and print, because the error status
    register is automatically cleared after reading in UMC ecc irq
    callback. So add such message printing in UMC ecc irq cb to be
    consistent with other RAS error interrupt cases.
    
    Signed-off-by: Guchun Chen <guchun.chen@amd.com>
    Reviewed-by: Tao Zhou <tao.zhou1@amd.com>
    Signed-off-by: Alex Deucher <alexander.deucher@amd.com>

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_umc.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_umc.c
index 9dd51f0d2c11..8ffa015bc1dc 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_umc.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_umc.c
@@ -120,6 +120,9 @@ int amdgpu_umc_process_ras_data_cb(struct amdgpu_device *adev,
 
 	/* only uncorrectable error needs gpu reset */
 	if (err_data->ue_count) {
+		dev_info(adev->dev, "%ld uncorrectable errors detected in UMC block\n",
+			err_data->ue_count);
+
 		if (err_data->err_addr_cnt &&
 		    amdgpu_ras_add_bad_pages(adev, err_data->err_addr,
 						err_data->err_addr_cnt))